### ISSUE TITLE
Add direct bundle artefact to releases

### ISSUE THREAD

**Issue by tukusejssirs** [romcal/romcal#149]
I think that some people might want to have an option to download the `dist/romcal.bundle.min.js` from the releases.

**Comment by emagnier** [comment]
Something like that? 
https://github.com/romcal/romcal/releases

**Comment by emagnier** [comment]
In the same time I'll not encouraging using that, because using a package manager in a project is still a best practice.

**Comment by tukusejssirs** [comment]
Well, by _direct_ I meant not the source code in archives, but the file itself (not in an archive).

Also I prefer the `npm` way (either built or installed by `npm`), but if the directly accessible bundle file is accessible from the release page, one might get the latest release (using the GitHub API) fairly easily and they would be sure they are using the latest version.

**Comment by TobiTenno** [comment]
i agree with @emagnier on this, so much of this is automated, we'd need to make a release and upload to it during the release process, which would get pretty messy

**Comment by TobiTenno** [comment]
i'm rereading this again and i'm very confused by what you're actually asking, @tukusejssirs, as npm has the tarballed archive that you can download from npm at any time. are you wanting one of each of the tarballs?

**Comment by TobiTenno** [comment]
since the end goal is indecisive, i'm closing this as not planned for the time being
